Savanna/Model/Game.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace Savanna
+{
+    /// <summary>
+    /// Mechanics of the game.
+    /// </summary>
+    public class Game
+    {
+        /// <summary>
+        /// Array that will contain the game field.
+        /// </summary>
+        public string[,] GameField { get; set; }
+
+        /// <summary>
+        /// Starts the game with empty board.
+        /// </summary>
+        public Game()
+        {
+            GameField = new string[GameParameters.boardColumns, GameParameters.boardRows];
+            ClearBoard();
+        }
+
+        /// <summary>
+        /// Clears the board.
+        /// </summary>
+        private void ClearBoard()
+        {
+            for (int column = 0; column < GameParameters.boardColumns; column++)
+            {
+                for (int row = 0; row < GameParameters.boardRows; row++)
+                {
+                    GameField[column, row] = GameMessages.emptyCell;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Checks if cell is on the board.
+        /// </summary>
+        /// <param name="newXCoordinate">New X coordinate.</param>
```

Comment:
It is better not use generic x and y. Better to use row and column.

Reply:
Renamed.